web/src/redux/slices/TestDefinition.slice.ts

```diff
@@ -109,7 +69,23 @@ const testDefinitionSlice = createSlice<
         return def;
       });
     },
+    revertDefinition(state, {payload: {originalSelector, selector}}) {
+      const initialDefinition = state.initialDefinitionList.find(
+        definition => definition.originalSelector === originalSelector
+      );
+
+      state.definitionList = initialDefinition
+        ? state.definitionList.map(definition =>
+            definition.originalSelector === originalSelector ? initialDefinition : definition
+          )
+        : state.definitionList.filter(definition => definition.selector !== selector);
```

Comment:
Minor: I think we could do `definition.originalSelector !== originalSelector` here and avoid passing the `selector` to `revertDefinition`. But when adding a new assertion we also need to create the `originalSelector` field.

Reply:
Yeah, I was thinking the same. Let's me try to see how that goes